### PR TITLE
Frontend react

### LINKED ISSUE #8124: FRONTEND WITH FRAMEWORK: React 9.1: React icons
### Description

Description
Topics covered
- What is react icon
- Why to use it
- How to use it
- Different method of styling icons

### Domain

Frontend Dev React/Angular/Vue

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by its-me-Harsh-Anand**
<hr>

## Description 📜
Topics covered
    - What is react icon
    - Why to use it
    - How to use it
    - Different method of styling icons

Fixes #8124 

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev #React/Angular/Vue

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by its-me-Harsh-Anand** [comment]
@anandhu720 Please review this PR sir

File: Frontend_Web_Development_React_Angular_Vue/React/Week 9/react-icons.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+const Alarm = styled(BsFillAlarmFill)`
+  color: red;
+  transform: scale(2);
+  margin: 5%;
+`;
+```
+
+## Conclusion
+Phew! We have covered so many things. Now, it's time to apply all these things in your beautiful projects. Hopw you have learnt everything about React Icons.
+
```
Comment (anandhu720) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 9/react-icons.md:270]: As per GWOC Team you can't add your name in .md file.Also add Referance links

**Review by anandhu720 — CHANGES_REQUESTED** [review]

**Comment by anandhu720** [comment]
@its-me-Harsh-Anand Also please dont create issue by yourself.We are taking this one because you already created this pr.Check our milestone to see open issues which you can work on

File: Frontend_Web_Development_React_Angular_Vue/React/Week 9/react-icons.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+__Contributor__ : [Harsh Anand](https://www.github.com/its-me-Harsh-Anand)
```
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 9/react-icons.md:271]: Remove your name from the Documentation.

**Review by heet-kumar — CHANGES_REQUESTED** [review]
@its-me-Harsh-Anand It is against the GWOC instruction

File: Frontend_Web_Development_React_Angular_Vue/React/Week 9/react-icons.md (hunk 1 above)
Comment (its-me-Harsh-Anand) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 9/react-icons.md:270]: Actually sir, I was unaware of this as I'm new to contribute in this domain. But, in JavaScript domain, it was compulsary to write our name at the end of the README file. That's why I added. I'll remove it, sir

File: Frontend_Web_Development_React_Angular_Vue/React/Week 9/react-icons.md (hunk 2 above)
Comment (its-me-Harsh-Anand) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 9/react-icons.md:271]: Okay sir, I'll remove but it was compulsary to add name in JavaScript domain. So, I thought it is compulsary here too.

**Review by anandhu720 — APPROVED** [review]

**Pull request merged**